worker/run.py

```diff
@@ -191,6 +192,11 @@ def update_status_and_check_killed(status):
                     self._container_id = do_start()
                 else:
                     raise
+
+            self._image_manager.touch_image(self._resources['docker_image'])
```

Comment:
I originally used the digest then discovered that you can't remove images by their digest..

Maybe I'll see if I can do a reverse lookup by digest to a get a name that I can delete the image by when I need to.